src/main/resources/swagger/api-docs.yaml

```diff
@@ -3384,6 +3407,99 @@ definitions:
         type: boolean
         description: true if you have access to the dataset, false otherwise
 
+  EditMethodRequest:
+    description: info to edit a method
+    type: object
+    properties:
+      source:
+        description: ID of base method to edit
+        type: object
+        properties:
+          namespace:
+            type: string
+            description: Method namespace
+          name:
+            type: string
+            description: Method name
+          snapshotId:
+            type: integer
+            default: 0
+            description: Method version
+        required:
+          - namespace
+          - name
+          - snapshotId
+      synopsis:
+        description: synopsis for new snapshot
+        type: string
+      documentation:
+        description: documentation for new snapshot
+        type: string
+      payload:
+        description: WDL for new snapshot
+        type: string
+      redactOldSnapshot:
+        description: whether or not to redact the source snapshot upon success
+        type: boolean
+    required:
+      - source
+      - synopsis
+      - documentation
+      - payload
+      - redactOldSnapshot
+
+  EditMethodResponse:
+    description: response of an edit method operation
+    type: object
+    properties:
+      method:
```

Comment:
can't we just reference an existing schema def, something like:
```
      method:
        $ref: '#/definitions/MethodResponse'
```
instead of manually listing out all the properties?

(not sure that's the right definition to reference)